File: qe-selenium/qe-selenium-java/src/main/java/com/demos/Activity2.java
package com.demos;

import org.openqa.selenium.By;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity2 {
    public static void main(String[] args) {
        FirefoxDriver driver=new FirefoxDriver();
        driver.get("https://training-support.net/webelements/login-form/");
        driver.findElement(By.xpath("//*[@id=\"username\"]")).sendKeys("admin");
        driver.findElement(By.xpath("//*[@id=\"password\"]")).sendKeys("password");
        driver.findElement(By.xpath("//button[@class=\"svelte-1pdjkmx\"]")).click();
        driver.close();

    }
}

File: qe-selenium/qe-selenium-python/Activity2.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:
    driver.get('https://training-support.net/webelements/login-form/')
    print(driver.title)
    driver.find_element(By.XPATH,"//*[@id=\"username\"]").send_keys('admin')
    driver.find_element(By.XPATH,"//*[@id=\"password\"]").send_keys('password')
    driver.find_element(By.XPATH,"/html/body/div/main/div/div/div/div/div[2]/form/button").click()


    driver.quit()



# from selenium.webdriver.common.by import By

# from selenium import webdriver
# with webdriver.Firefox() as driver:
#     driver.get('https://training-support.net/webelements/login-form/')
#     print(driver.title)
#     driver.find_element(By.XPATH,"//input[@id=\"username\"]").send_keys('admin')
#     driver.find_element(By.XPATH,"//input[@id=\"password\"]").send_keys('password')
#     driver.find_element(By.XPATH,"//button[@class=\"svelte-1pdjkmx\"]").click()

#     print(driver.title)
#     driver.quit()
